db/changes.go

```diff
@@ -682,9 +700,9 @@ func (db *Database) SimpleMultiChangesFeed(ctx context.Context, chans base.Set,
 		for {
 			// Updates the ChangeWaiter to the current set of available channels
 			if changeWaiter != nil {
-				changeWaiter.UpdateChannels(channelsSince)
+				changeWaiter.UpdateChannels(activeChannels)
 			}
-			base.DebugfCtx(ctx, base.KeyChanges, "MultiChangesFeed: channels expand to %#v ... %s", base.UD(channelsSince.String()), base.UD(to))
+			base.DebugfCtx(ctx, base.KeyChanges, "MultiChangesFeed: channels expand to %#v ... %s", base.UD(activeChannels.ToSerializedStrings()), base.UD(to))
```

Comment:
If I recall correctly, the work to build activeChannels.ToSerializedStrings() is going to occur even if we don't have debug logging enabled.  Would be preferable to avoid this, but it would mean an enhancement to UD to support channel.Set in a similar way that we're supporting base.Set with RedactorSet.  Some work would be needed to avoid a circular dependency there (probably by implementing a redaction interface in base that both base.Set and channels.Set can interface, then type switch on that interface).  
I'm fine leaving that as a follow-on enhancement, but can you file a ticket for that to make sure it's done?

Reply:
I implemented this with the `GoString` method which will do lazy evaluation of it and removes `ToSerializedStrings`